package.json

```diff
@@ -2,9 +2,9 @@
 	"name": "@lifesg/validation-schema-generator",
 	"version": "1.0.0-alpha.1",
 	"description": "JSON to validation schema for frontend-engine",
-	"main": "dist/cjs/index.js",
-	"module": "dist/index.js",
-	"typings": "dist/index.d.ts",
+	"main": "cjs/index.js",
```

Comment:
Eh why is this so?

Reply:
because `post-build.js` script is removed and there is a need to remove the `dist/` path when copied over to the `dist/` folder

instead of doing this way, I added the `dist/` folder in the rollup config so the bundling can still run normally